src/autora/doc/pipelines/main.py

```diff
@@ -15,6 +18,27 @@
     format="%(asctime)s %(levelname)s %(module)s.%(funcName)s(): %(message)s",
 )
 logger = logging.getLogger(__name__)
+nltk.download("wordnet")
+
+
+def evaluate_documentation(predictions: List[List[str]], references: List[str]) -> Tuple[float, float]:
+    # Tokenize predictions and references
+    tokenized_predictions = [pred[0].split() if pred else [] for pred in predictions]
+    tokenized_references = [[ref.split()] for ref in references]
+
+    # Calculate BLEU score
+    bleu = corpus_bleu(
```

Comment:
Can you add a comment about why we are using `method1`?

Reply:
Done!